test/provider_os_memory.cpp

```diff
@@ -22,20 +22,6 @@ typedef enum purge_t {
     PURGE_FORCE = 2,
 } purge_t;
 
-static umf_os_memory_provider_params_t UMF_OS_MEMORY_PROVIDER_PARAMS_TEST = {
-    /* .protection = */ UMF_PROTECTION_READ | UMF_PROTECTION_WRITE,
-    /* .visibility = */ UMF_VISIBILITY_PRIVATE,
-
-    // NUMA config
-    /* .nodemask = */ NULL,
-    /* .maxnode = */ 0,
-    /* .numa_mode = */ UMF_NUMA_MODE_DEFAULT,
-    /* .numa_flags = */ 0,
-
-    // others
-    /* .traces = */ 1,
```

Comment:
traces set here to 1 (not sure if relevant, though). Please, confirm it's not just omitted by mistake

Reply:
It was intentional. Some ENV variable should control the traces and we should not hardcode it in tests to 1.